makisu/makisu.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: tekton.dev/v1alpha1
+kind: Task
+metadata:
+  name: makisu
+spec:
+  inputs:
+    params:
+    - name: CONTEXTPATH
+      description: Path to the build context.
+      default: /workspace
+    - name: PUSH_REGISTRY
+      description: Registry to push image to.
+      default: index.docker.io
+    - name: REGISTRY_SECRET
+      description: Secret containing information about the used regsitry.
+      default: docker-registry-config
+
+    resources:
+    - name: source
+      type: git
+
+  outputs:
+    resources:
+    - name: image
+      type: image
+
+  steps:
+  - name: build-and-push
+    image: gcr.io/makisu-project/makisu:v0.1.9
```

Comment:
If you want you can set the image tag to `v0.1.10` (see: https://github.com/uber/makisu/releases/tag/v0.1.10) it seems like there are no breaking changes (at least in the parts that are used here).